storage_service/locations/models/event.py

```diff
@@ -117,12 +137,28 @@ class Meta:
         verbose_name = _("Callback")
         app_label = "locations"
 
-    def execute(self, url=None):
+    def get_event_label(self):
```

Comment:
They builtin `get_event_display` should cover this case. See: https://docs.djangoproject.com/en/1.8/ref/models/instances/#django.db.models.Model.get_FOO_display

Reply:
Oh, that's nice!